README.md

```diff
@@ -10,30 +10,34 @@ To get up and running quickly, see our [installation tutorial][tutorial].
 
 ### Requirements
 
-- [Klotho CLI](https://klo.dev/docs/tutorials/getting_started_with_klotho#installing-the-cli)
+- [Klotho CLI](https://klo.dev/docs/tutorials/built_with_klotho/slackbot)
 - Node.js 14.x+ (& NPM)
 
 ### Building
 
-Just once (or whenever we add new dependencies):
-
 ```bash
-npm install
+npm run klotho build
 ```
 
-Then, to build:
-
-```bash
-npm run klotho:build
-```
+This will compile the TypeScript, run the tests, and run Klotho on the application.
 
 ### Deploying
 
-Assuming you [have your Pulumi stack configured][config]:
+1. Just once, run:
 
-```bash
-npm run klotho:deploy
-```
+   ```bash
+   npm run klotho pulumi config set aws:region <YOUR DESIRED REGION> # e.g. us-east-1
+   ```
+2. Make sure you have your env set up with AWS credentials (see [our tutorial][tutorial-aws].
+3. ```bash
+   npm run klotho pulumi up
```

Comment:
I dislike `klotho pulumi` - we're not involved in this part and this gives the wrong impression.

Reply:
Do you dislike the specific invocation, or the idea in general?

I've found it's pretty handy to have a command that "just works", including (a) finding the right `compiled*/` dir that klotho created, and (b) finding the stack name. With these commands, I just export or unset `KLOTHO_APP_NAME` and I'm good to go.

(I think we are at least a bit involved: we created the dir with the pulumi contents, and seeded the stack.)

Reply:
btw, I originally had this as specific commands:

```
npm run klotho up
npm run klotho config <set ...>
npm run klotho down
```

... but all of these were just invoking pulumi with the dir + stack, so I thought it's more straightforward to do that.

Reply:
I would prefer it being `npm run pulumi up`. I don't want to give the impression that either a) We = Pulumi b) They can get Pulumi support from us c) That we are replacing/wrapping Pulumi. We're involved in the generation of the files, but that's the boundary. We're "slotting" into a release process, but not at the deployment steps - we're pre-deployment. We can discuss as a group